README.md

```diff
@@ -10,4 +10,31 @@ bazel test //...
 
 ## Developing Entangled 
 - Be sure to run `./hooks/autohook.sh install` after initial checkout!
-- Pass `-c dbg` for building with debug symbols.
\ No newline at end of file
+- Pass `-c dbg` for building with debug symbols.
+
+### Tools required for running git commit hook
+- buildifier
+- clang-format
+
+### Buildifier
+Buildifier can be installed with `bazel` or `go`
+
+#### Install with go
+1. change directory to `$GOPATH`
+2. run `$ go get github.com/bazelbuild/buildtools/buildifier`
+   The executable file will be located under `$GOPATH/bin`
+3. make a soft link for global usage, run 
```

Comment:
Instead of creating symbolic link, would it be more straightforward to copy `buildifier` executable to `/usr/bin` or `/usr/local/bin`?